internal/transform/claimable_balance.go

```diff
@@ -36,7 +36,7 @@ func TransformClaimableBalance(ledgerChange ingest.Change) (ClaimableBalanceOutp
 		return ClaimableBalanceOutput{}, fmt.Errorf("Invalid balanceId in op: %d", uint32(ledgerEntry.LastModifiedLedgerSeq))
 	}
 	outputFlags := uint32(balanceEntry.Flags())
-	outputAsset, err := transformSingleAsset(balanceEntry.Asset)
+	outputAsset, err := transformSingleAsset(balanceEntry.Asset, LedgerCloseMeta)
```

Comment:
I don't think it makes sense to extract the ledger close timestamp in this method. The timestamp is tied to the ledger close, not asset. Is it possible to extract the timestamp separately? I wouldn't expect the `transformSingleAsset` to return the ledger closed timestamp without good reason.